utils/generate_js_fixtures.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import subprocess
+from pathlib import Path
+
+SPHINX_ROOT = Path(__file__).resolve().parents[1]
+TEST_JS_FIXTURES = SPHINX_ROOT / 'tests' / 'js' / 'fixtures'
+TEST_JS_ROOTS = SPHINX_ROOT / 'tests' / 'js' / 'roots'
+
+
+def build(srcdir: Path) -> None:
+    cmd = ('sphinx-build', '-E', '-q', '-b', 'html', f'{srcdir}', f'{srcdir}/build')
```

Comment:
Should we really make it in build or in _build instead?

Reply:
Probably better to use `_build` indeed.  Updated (including the `.gitignore` file.  there may be some unrelated stale entries in there to clean up separately).